src/main/java/frc/robot/Constants.java

```diff
@@ -5,6 +5,15 @@
 package frc.robot;
 
 import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.geometry.Translation3d;
+import edu.wpi.first.math.Pair;
+import edu.wpi.first.math.geometry.Rotation3d;
+import edu.wpi.first.math.geometry.Transform3d;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.photonvision.PhotonCamera;
 import edu.wpi.first.math.util.Units;
```

Comment:
Is this another git bulldozer situation where the both sides of a merge conflict were accepted? Small thing but maybe this import can be moved up with the other `edu.wpi` imports.